ajax/libs/jquery.x18n/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "jquery.x18n",
+  "filename": "jquery.x18n.min.js",
+  "version": "1.1.3",
+  "description": "The jQuery adapter for the x18n internationalisiation library.",
```

Comment:
Seems like `internationalisiation` from upstream is a mistake of "internationalisation". In description of [README.md](https://github.com/florian/jQuery.x18n#jqueryx18n-) the latter one is used.